.github/workflows/deploy-formstack.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+env:
+  DIRECTORY_NAME: formstack-baton-requests
+
+jobs:
+  CI:
+    name: Deploy formstack-baton-requests
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'corretto'
+          cache: 'sbt'
+      - name: Build App
+        working-directory: ${{env.DIRECTORY_NAME}}
+        run: sbt clean compile riffRaffUpload
+      - name: Upload to Riff Raff
```

Comment:
Doesn't `riffRaffUpload` in the previous line upload to Riffraff?  I'm asking as I've never used `actions-riff-raff` so I'm a bit confused

Reply:
If it were me, I'd just leave off the `riffRaffArtifactResources`, and then get rid of the sbt plugin for riff raff.

Reply:
Ah I see.  So they both do the same thing but one's through SBT and the other's through GHA.

Reply:
Yeah exactly. Little bit of a faff in github actions if you're not aware of whether it needs to produce a .jar or .deb file, and work out where it needs to land in the riffRaff folder structure, but hopefully once converted from TC to GHA, it'll be more clear to those who come to a project afresh